weather_mv/loader_pipeline/bq.py

```diff
@@ -94,6 +95,9 @@ class ToBigQuery(ToDataSink):
     skip_region_validation: bool
     disable_grib_schema_normalization: bool
     coordinate_chunk_size: int = 10_000
+    should_create_polygon: bool = False
+    lat_grid_resolution: t.Optional[float] = None
+    lon_grid_resolution: t.Optional[float] = None
 
     @classmethod
     def add_parser_arguments(cls, subparser: argparse.ArgumentParser):
```

Comment:
Let's add a flag in the subparser to let users turn this feature on or off.

Reply:
Code updated as per your suggestion. Thanks for the input!